tools/RAiDER/cli/raider.py

```diff
@@ -209,8 +211,8 @@ def calcDelays(iargs=None):
     aoi = params['aoi']
     model = params['weather_model']
 
-    # add a small buffer
-    aoi.add_buffer(buffer = 1.5 * model.getLLRes())
+    # adjust user requested AOI by grid size and buffer slightly
+    aoi.add_buffer(model.getLLRes())
```

Comment:
I am unclear why you modify this to be only 1* . Some of these resolutions are approximate and ensuring its >1 but technically <2 would make sure there is at least a grid cell added.

Reply:
I agree with @dbekaert  here. You probably want at least small amount >1.

Reply:
sorry this was a typo